docs/community_resources/release_process.md

```diff
@@ -1,3 +1,7 @@
+---
+description: "Ibexa DXP releases new versions periodically in three flavors: Ibexa Content, Ibexa Experience and Ibexa Commerce, plus open-source Ibexa OSS."
```

Comment:
```suggestion
description: "Ibexa DXP releases new versions periodically in different flavors: Ibexa Content, Ibexa Experience and Ibexa Commerce, plus open-source Ibexa OSS."
```
Forgot to add in PR decription: quotation marks are required when description contains a colon, otherwite the yaml formatting is broken